pkg/processor/trigger/nats/test/nats_test.go

```diff
@@ -86,6 +86,7 @@ func (suite *testSuite) getDeployOptions() *platform.CreateFunctionOptions {
 		Attributes: map[string]interface{}{
 			"topic": suite.topicName,
 		},
+		MaxWorkers: 10,
```

Comment:
use golang function then. 10 python interpreters will choke small ci builders

Reply:
it was just for testing quickly, 3 is also enough.